matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt

```diff
@@ -62,15 +62,18 @@ data class TimelineEvent(
     }
 
     fun getDisambiguatedDisplayName(): String {
-        return if (isUniqueDisplayName) {
+        val disambiguated = if (isUniqueDisplayName) {
             senderName
         } else {
             senderName?.let { name ->
                 "$name (${root.senderId})"
             }
         }
-               ?: root.senderId
-               ?: ""
+        return if (disambiguated.isNullOrBlank()) {
+            root.senderId ?: ""
+        } else {
+            disambiguated
```

Comment:
If display name is not unique and senderName is blank, the result will be " (@bob:matrix.org)" and should be "@bob:matrix.org"

Reply:
Will fix, thanks!